terraform/deployments/variables/test/apps.tfvars

```diff
@@ -2,7 +2,7 @@
 # App variables
 #--------------------------------------------------------------
 
-app_domain          = "test.publishing.service.gov.uk"
+app_domain          = "test.govuk.digital"
```

Comment:
Perhaps you could add a TODO comment here to explain why we're using this domain?

```diff
- app_domain          = "test.publishing.service.gov.uk"
+ app_domain          = "test.govuk.digital" # TODO: Changed until we have router running in ECS
```

Reply:
ok, I will add a comment.